doc/user/content/release-notes.md

```diff
@@ -49,57 +49,174 @@ Wrap your release notes at the 80 character mark.
 <span id="v0.4.1"></span>
 ## v0.4.1 (Unreleased)
 
-- Make casting from `numeric` and `float` to `int` consistent with PostgreSQL by
-  rounding before the conversion.
+No release notes yet.
 
 <span id="v0.4.0"></span>
 ## v0.4.0
 
-- Support for SASL PLAIN Authentication to support Confluent Cloud {{% gh 3418 %}}
-- Update Change Data Capture format to allow Kafka sinks to optionally emit Debezium
-  style [consistency metadata](/sql/create-sink/#consistency-metadata).
-- Introduce the ability to [rename indexes, sinks, sources, and
-  views](/sql/alter-rename).
-- Rename the `-w`/`--threads` command line argument to `-w`/`--workers`, since it
-  reflects timely workers and does not limit the number of threads that materialized may
-  start.
-- Fix a bug that prevented dropping databases with cross-schema dependencies.
-  {{% gh 3558 %}}
-- Expose [Prometheus metrics for sinks](https://materialize.io/docs/monitoring/).
-- Allow column names in SELECT clause to be used by GROUP BY {{% gh 1673 %}}
-- Make decimal / float to int casts behave similar to Postgres {{% gh 3700 %}}
-- Fix a bug that prevented ordering by columns that are not in the select clause {{% gh
-  696 %}}
-- Support SHOW TRANSACTION ISOLATION LEVEL {{% gh 800 %}}
-- Support to_jsonb(sql record) {{% gh 2414 %}}
-- Add timestamp to Sink CDC field {{% gh 3216 %}}
+- Rename the `--threads` command-line option to [`--workers`](/cli/#worker-threads),
+  since it controls only the number of dataflow workers that Materialize will
+  start, not the total number of threads that Materialize may use. The short
+  form of this option, `-w`, remains unchanged.
+  **Backwards-incompatible change.**
+
+- Add the `--experimental` command-line option to enable a new [experimental
+  mode](/cli/#experimental-mode), which grants access to experimental features
+  at the risk of compromising stability and backwards compatibility. Forthcoming
+  features that require experimental mode will be marked as such in their
+  documentation.
+
+- Support [SASL PLAIN authentication for Kafka sources](/sql/create-source/avro-kafka/#connecting-to-a-kafka-broker-using-sasl-plain-authentication).
+  Notably, this allows Materialize to connect to Kafka clusters hosted by
+  Confluent Cloud.
+
+- Do not require [Kafka Avro sources](sql/create-source/avro-kafka) that use
+  `ENVELOPE NONE` or `ENVELOPE DEBEZIUM` to have key schemas whose fields are a
+  subset of the value schema {{% gh 3677 %}}.
+
+- Teach Kafka sinks to emit Debezium style [consistency
```

Comment:
Is there a way to easily insert CDC or Change Data Capture back into this sentence?  I was hoping to make that keyword more visible, in case someone skims the relnote really quickly, and doesn't connect the dots between Debezium and CDC.  

But not a big deal, only if it's easy

Reply:
Ack, added a sentence about this. I'm sorta confused about the wording we're using around this, so did my best to link them together—I think the technically precise way is to call the combination of the metadata and data topics together the "CDC format".

Reply:
thanks!